skorch/classifier.py

```diff
@@ -231,6 +231,12 @@ def predict(self, X):
         """
         return self.predict_proba(X).argmax(axis=1)
 
+    def __sklearn_tags__(self):
+        # TODO: this is just the bare minimum, more tags should be added
+        tags = super().__sklearn_tags__()
+        tags.estimator_type = 'classifier'
```

Comment:
you should probably be inheriting from `ClassifierMixin` instead really.

Reply:
Right now, we don't inherit from `BaseEstimator`, `ClassifierMixin`, etc. So far, this went well but probably can cause issues like this. Would you think it is better to switch?

Reply:
But you're already inherriting from `ClassifierMixin` in this file, so adding `BaseEstimator` would only make things easier.

In general, we've added a substantial amount of features to the `BaseEstimator`, and you probably would have a much easier time if you inherit from it. These days I'd strongly recommend inheriting from the right mixins.